app/routes/index.js

```diff
@@ -14,7 +14,29 @@ router.get('/login', async function(req, res, next){
 });
 
 router.post('/login', async function(req, res, next){
-  // TODO: fill in login logic
+  try {
+
+    const user = req.body;
+    const user_in_db = await usersModel.getByEmail(user.email);
+
+    if(user_in_db.length === 0){
```

Comment:
Codacy has a fix for the issue: [Expected space(s) after "if".](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443778)
```suggestion
    if (user_in_db.length === 0){
```